react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
 
 {{% read_before_2 "This tutorial is part 3 of 3 in this series." "Part 1:" "A complete React with GraphQL Tutorial" "https://www.robinwieruch.de/react-with-graphql-tutorial/" "Part 2:" "A complete React with Apollo and GraphQL Tutorial" "https://www.robinwieruch.de/react-graphql-apollo-tutorial/" %}}
 
-In your previous applications, you may have used a GraphQL client library, which is potentially partly view layer agnostic, but can be used with React or other view layer solutions. There are a couple of those GraphQL client libraries out there: Apollo Client, Urql and Relay. In the following, I want to show you that it isn't too difficult to implement your own GraphQL client library that works with React. However, the takeaway shouldn't be "build your own GraphQL client for your production ready applications" but rather:
+You may have used a GraphQL client library that was at least partly view layer agnostic, but still able to work with React or other view layer solution in the form of applications like Apollo Client, Urql and Relay. The next chapter is to illustrate how to implement a GraphQL client library that works with React. The takeaway shouldn't be "build your own GraphQL client for your production ready applications", however, you should instead learn:
 
-* see how a GraphQL client library works under the hood
-* see how it isn't too difficult to implement it your way
-* see how you would be able to contribute to the GraphQL ecosystem
+* how a GraphQL client library works under the hood
+* how simple a GraphQL client library is to implement your way
+* how it is capable of contributing to the GraphQL ecosystem
 
-I believe there is great potential to contribute to the GraphQL ecosystem, because it is fairly new. It would be great to have a diverse set of tools out there and not only one ecosystem pushing the whole thing forward. So see it as your opportunity to contribute to it.
+There is contribute to the GraphQL ecosystem, because the tools surrounding it are still fairly new. A diverse set of tools would speed this along, instead of Apollo pushing its maturation forward alone. This is not only a useful addition for your web development skillset, it is also an opportunity to contribute to the early stages of GraphQL.
```

Comment:
"There is contribute to the GraphQL ecosystem ..." -> "There is a great potential to contribute to the GraphQL ecosystem ..."

Reply:
Changed to "There is lots of room to contribute".  The phrase "great potential" just doesn't sound right in discussing software.  More appropriate for a sports prospect.  The word "great" is considered hyperbole in English, which we should try to avoid (words like amazing, wonderful, etc.).

Reply:
Sounds good!